apps/backend/api/graphql/mutations/fundingRound.js

```diff
@@ -0,0 +1,236 @@
+/* global FundingRound, Group, GroupMembership, Responsibility, Queue, bookshelf, Post, FundingRoundUser, PostUser */
+import { } from 'lodash'
+import { GraphQLError } from 'graphql'
+import convertGraphqlData from './convertGraphqlData'
+
+// XXX: because convertGraphqlData messes up dates
+const fixDateFields = (attrs, data) => {
+  const dateFields = [
+    { from: 'publishedAt', to: 'published_at' },
+    { from: 'votingOpensAt', to: 'voting_opens_at' },
+    { from: 'votingClosesAt', to: 'voting_closes_at' },
+    { from: 'submissionsOpenAt', to: 'submissions_open_at' },
+    { from: 'submissionsCloseAt', to: 'submissions_close_at' }
+  ]
+  dateFields.forEach(({ from, to }) => {
+    if (data[from]) {
+      attrs[to] = new Date(Number(data[from]))
+    }
+  })
+  return attrs
+}
+
+export async function createFundingRound (userId, data) {
+  const attrs = convertGraphqlData(data)
+  // Required fields
+  if (!attrs.title) throw new GraphQLError('title is required')
+  if (!attrs.group_id) throw new GraphQLError('groupId is required')
+
+  const group = await Group.find(attrs.group_id)
+  if (!group) throw new GraphQLError('Invalid group')
+
+  const canManage = await GroupMembership.hasResponsibility(userId, group, Responsibility.constants.RESP_MANAGE_ROUNDS)
```

Comment:
I'm guessing its defined elsewhere, that I haven't seen yet but since this change-set is 120 files, I'll ask here...

I am assuming that this new resp is added to the steward common role (or whichever is the common role that has everything)?

Reply:
yes